data/versions.yml

```diff
@@ -177,7 +177,7 @@
 - version: "2.02"
   date: "2023-03-15"
   changes:
-    - Reintroduced the user [OAuth flow](#oauth2)
+    - Reintroduced the user [OAuth flow](#api-authorization)
     - Oauth token expires after 2 hours
```

Comment:
I would say we should link it to #api-authorization-oauth-flow which we reintroduced precisely, as #api-authorization covers the general token